recipes/tryingsnake/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "tryingsnake" %}
+{% set version = "0.4.0.dev1" %}
```

Comment:
This looks like it is prerelease. Since the main conda forge channel is for stable releases, you can either make this release official, or use the `prerelease` channel by adding it in `conda_build_config.yaml`.

Reply:
Thank you for your review @npavlovikj 

To be honest I wanted to draft a release, and update project README to reflect new install option, before making the release and signing corresponding tag. 

I guess not shortcuts here... :)